docs/sections/user_guide/cli/mode_config.rst

```diff
@@ -138,7 +367,54 @@ TBD
 Examples
 ~~~~~~~~
 
-TBD
+The examples that follow use atparse-formatted template file ``atparse.txt`` with content
+
+.. code:: sh
+  @[greeting], @[recipient]!
+
+* Convert an atparse-formatted template file to Jinja2 format:
+
+  .. code:: sh
+
+    $ uw config translate --input-file atparse.txt --input-format atparse --output-format jinja2
```

Comment:
I was pondering that a bit - the cleaner UX is to default it and check at runtime, though requiring it could have an argument for reinforcing user awareness

Reply:
Will `uw config translate` ever do anything but atparse -> Jinja2 translation? If so, I think we should keep things as they are so that we won't have to break backward-compatibility when we add another capability. It seems possible that some other kind of translation might be desired in the future, though [UW-410](https://jira.epic.oarcloud.noaa.gov/browse/UW-419) concerns might limit what we can do.

If we'll _never_ do anything but atparse -> Jinja2, maybe the submode has too general a name. I can't think of anything I like at the moment. `uw config atparse-to-jinja2` is kind of awful, if clear.

Reply:
I think it could be 6 of one....leaving it as-is would be fine. Just felt sort of verbose since that's the only thing it does and we don't have plans to expand it.

Reply:
Gosh I hope not - I think `translate` is sufficient, but I think we're in agreement that locking out the formats is a good refactor?

Reply:
I'd vote for opening a Jira ticket describing the proposal, and we can discuss / define acceptance criteria / pull into a sprint during some near-future sprint planning.

Reply:
Opened as UW-472